### ISSUE TITLE
guides: "Start tutorial" link is somewhat lost because of lack of colour contrast

### ISSUE THREAD

**Issue by myitcv** [play-with-go/play-with-go#231]
<img width="1131" alt="Screen Shot 2020-11-11 at 12 19 12" src="https://user-images.githubusercontent.com/3374574/98811090-2ac06900-2418-11eb-8038-54c41af4e360.png">

This is something of an accessibility issue, but could also be a problem for first-time users.

**Comment by marcosnils** [comment]
Fixed by a4d42cecb2986d13b7e01621bd0b4cabb2e04a54
